nautobot_ssot/contrib/model.py

```diff
@@ -168,10 +171,18 @@ def _handle_single_field(
             else:
                 related_object_content_type = relationship.destination_type
             related_model_class = related_object_content_type.model_class()
-            relationship_fields["custom_relationship_many_to_many_fields"][field] = {
-                "objects": [diffsync.get_from_orm_cache(parameters, related_model_class) for parameters in value],
-                "annotation": custom_relationship_annotation,
-            }
+
+            if relationship.type == RelationshipTypeChoices.TYPE_ONE_TO_MANY:
+                relationship_fields["custom_relationship_foreign_keys"][related_model_class.__name__] = {
```

Comment:
```suggestion
                relationship_fields["custom_relationship_foreign_keys"][field] = {
```

I think even though this is not used we should keep using the field name on the Pydantic object as the key, what do you think?